lib/src/Uncertainty/Algorithm/Simulation/openturns/NAISResult.hxx

```diff
@@ -0,0 +1,74 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief class for NAISResult
+ *
+ *  Copyright 2005-2020 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_NAISRESULT_HXX
+#define OPENTURNS_NAISRESULT_HXX
+
+#include "openturns/ProbabilitySimulationResult.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class NAISResult
+ *
+ * NAISResult is some NAISResult type to illustrate how to add some classes in Open TURNS
+ */
+class OT_API NAISResult
+  : public ProbabilitySimulationResult
+{
+
+  CLASSNAME
+public:
+
+  /** Default constructor */
+  NAISResult();
+
+  /** Get Samples */
+  Sample getSample() const;
```

Comment:
this is the last sample of the instrumental distribution (input), you want a more descriptive name:
getInstrumentalSample ?